src/App.js

```diff
@@ -39,19 +39,21 @@ class App extends Component {
     console.log(this.state);
   }
 
-  onItemClicked = () => {
-    // Fill this in for Wave 3!
+  increasePoint = () => {
+    this.setState({points: this.state.points + 1});
   }
 
   render() {
     const items = this.state.items.map((item, i) => {
       return <GameItem
-               height={item.height}     // Height - used for a CSS style to position on the screen
-               layer={100 + i}          // Layer - used for a CSS style to show items on-top of bg
-               key={item.id}            // Key - to help React with performance
+              height={item.height}     // Height - used for a CSS style to position on the screen
+              layer={100 + i}          // Layer - used for a CSS style to show items on-top of bg
+              key={item.id}            // Key - to help React with performance
 
                // Additional props (event callbacks, etc.) can be passed here
-             />;
+              type={item.type}
+              onItemClicked={this.increasePoint}
```

Comment:
Make sure that you are highlighting what the purpose of that function is with its name. If it’s a callback, use the term callback, if it’s an event handler, tell us what kind of event it is. Here `onItemClicked` is a callback function.